<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gauss</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>1311</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Uni</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1281</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Schule</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Sprache</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Skype</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1329</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMembers">
    <property name="title">
     <string>Members</string>
    </property>
    <widget class="QMenu" name="menuStudents">
     <property name="title">
      <string>Students</string>
     </property>
     <addaction name="actionAdd_Student"/>
    </widget>
    <widget class="QMenu" name="menuTeachers">
     <property name="title">
      <string>Teachers</string>
     </property>
     <addaction name="actionAdd_Teacher"/>
     <addaction name="actionShow_Teacher"/>
    </widget>
    <addaction name="menuStudents"/>
    <addaction name="menuTeachers"/>
   </widget>
   <widget class="QMenu" name="menuUtilities">
    <property name="title">
     <string>Utilities</string>
    </property>
    <widget class="QMenu" name="menuReceipts">
     <property name="title">
      <string>Receipts</string>
     </property>
     <addaction name="actionGive_Receipt"/>
     <addaction name="actionTake_Money"/>
    </widget>
    <addaction name="actionSchwer_Levels"/>
    <addaction name="actionManage_Departments"/>
    <addaction name="actionManage_Courses"/>
    <addaction name="actionManage_Fee_Schemes"/>
    <addaction name="actionManage_Pay_Schemes"/>
    <addaction name="actionManage_Instituts"/>
    <addaction name="actionManage_Langugages"/>
    <addaction name="actionManage_Kassen"/>
    <addaction name="actionManage_Diplomas"/>
    <addaction name="menuReceipts"/>
   </widget>
   <widget class="QMenu" name="menuStatistics">
    <property name="title">
     <string>Statistics</string>
    </property>
    <addaction name="actionTest_Plot"/>
    <addaction name="actionMachine_Learning"/>
   </widget>
   <widget class="QMenu" name="menuGroups">
    <property name="title">
     <string>Groups</string>
    </property>
    <addaction name="actionCreate_Uni_Group"/>
    <addaction name="actionCreate_Schule_Group"/>
    <addaction name="actionCreate_Sprache_Group"/>
    <addaction name="actionBuildings_and_Rooms"/>
    <addaction name="actionAdd_Requests"/>
    <addaction name="actionShow_All_Groups"/>
   </widget>
   <addaction name="menuMembers"/>
   <addaction name="menuUtilities"/>
   <addaction name="menuStatistics"/>
   <addaction name="menuGroups"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Teacher"/>
   <addaction name="actionAdd_Student"/>
  </widget>
  <action name="actionSchwer_Levels">
   <property name="text">
    <string>Schwer Levels</string>
   </property>
  </action>
  <action name="actionAdd_Student">
   <property name="icon">
    <iconset resource="../piges.qrc">
     <normaloff>:/icons/icons/student.png</normaloff>:/icons/icons/student.png</iconset>
   </property>
   <property name="text">
    <string>Add Student</string>
   </property>
  </action>
  <action name="actionAdd_Teacher">
   <property name="icon">
    <iconset resource="../piges.qrc">
     <normaloff>:/icons/icons/professor.png</normaloff>:/icons/icons/professor.png</iconset>
   </property>
   <property name="text">
    <string>Add Teacher</string>
   </property>
  </action>
  <action name="actionManage_Departments">
   <property name="text">
    <string>Manage Departments</string>
   </property>
  </action>
  <action name="actionManage_Courses">
   <property name="text">
    <string>Manage Courses</string>
   </property>
  </action>
  <action name="actionManage_Pay_Schemes">
   <property name="text">
    <string>Manage Pay Schemes</string>
   </property>
  </action>
  <action name="actionManage_Fee_Schemes">
   <property name="text">
    <string>Manage Fee Schemes</string>
   </property>
  </action>
  <action name="actionManage_Instituts">
   <property name="text">
    <string>Manage Instituts</string>
   </property>
  </action>
  <action name="actionManage_Langugages">
   <property name="text">
    <string>Manage Langugages</string>
   </property>
  </action>
  <action name="actionManage_Kassen">
   <property name="text">
    <string>Manage Kassen</string>
   </property>
  </action>
  <action name="actionCreate_Uni_Group">
   <property name="text">
    <string>Create Uni Group</string>
   </property>
  </action>
  <action name="actionCreate_Schule_Group">
   <property name="text">
    <string>Create Schule Group</string>
   </property>
  </action>
  <action name="actionCreate_Sprache_Group">
   <property name="text">
    <string>Create Sprache Group</string>
   </property>
  </action>
  <action name="actionBuildings_and_Rooms">
   <property name="text">
    <string>Buildings and Rooms</string>
   </property>
  </action>
  <action name="actionAdd_Requests">
   <property name="text">
    <string>Add Requests</string>
   </property>
  </action>
  <action name="actionShow_Teacher">
   <property name="text">
    <string>Show Teacher</string>
   </property>
  </action>
  <action name="actionTest_Plot">
   <property name="text">
    <string>test Plot</string>
   </property>
  </action>
  <action name="actionMachine_Learning">
   <property name="text">
    <string>Machine Learning</string>
   </property>
  </action>
  <action name="actionManage_Diplomas">
   <property name="text">
    <string>Manage Diplomas</string>
   </property>
  </action>
  <action name="actionGive_Receipt">
   <property name="text">
    <string>Give Money</string>
   </property>
  </action>
  <action name="actionTake_Money">
   <property name="text">
    <string>Take Money</string>
   </property>
  </action>
  <action name="actionShow_All_Groups">
   <property name="text">
    <string>Show All Groups</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../piges.qrc"/>
 </resources>
 <connections/>
</ui>
